premake5.lua

```diff
@@ -652,11 +652,10 @@ workspace "SumatraPDF"
     language "C++"
     cppdialect "C++latest"
     regconf()
-    disablewarnings { "4838" }

```

Comment:
There should be no changes in this file. Maybe you need to rebase

Reply:
Why that ?
I introduced a dependency on MuPDF & `EngineFzUtil.h` in `SumatraUnitTests.cpp` that required those changes.

Reply:
Didn't realize it had to pull in mupdf.

But looking closer, `resolve_link` doesn't need mupdf so it shouldn't use any code from it to avoid pulling it in as a dependency.

To convert string to int / float, you can use `str::Parse()` `atoi/atof` from C library https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/atof-atof-l-wtof-wtof-l?view=msvc-160

Reply:
And it would probably have to be moved to a file other than `EngineFzUtil.cpp` There's no great place so can just as well go into `EngineBase.cpp`.

Reply:
BTW: no tests for this functions is fine with me too. It's not that complicated.

Reply:
OK, I'll move that function to `EngineBase.cpp`